CHARTER.md

```diff
@@ -1,51 +1,61 @@
+### Chair Participation
+
+Chairs are expected to remain actively involved in the community. A chair who does not participate in ART voting *without reason and notice* three times in a row forfeits the position and becomes a regular ART member automatically.
+
+
+## Proposals
+
+Proposals may only be made asychronously by submitting a pull request to this repository with a new file in the [`art-decisions/proposals`](art-decisions/proposals) directory. The provided [template file](art-decisions/proposals/template.md) may be used as a guideline for new proposals. Proposals may only be submitted by current ART members. If you would like to encourage discussion around a proposal before any voting starts, please submit a draft pull request instead, and mark it ready for review once you are comfortable that you will not make any changes to your proposal. Open pull requests can be viewed at the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) page.
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a 66% majority with a quorum of 66% of all ART members is needed for the proposal to pass.
+Each member of the ART is eligible to vote if they have been a member for at least 30 days. For any proposal that impacts the whole project or processes, **approval requires a minimum of four votes with 75% approval among those voting**.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 2 months after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+Any vote must be open for at least 7 calendar days to account for different time zones and other obligations and must be open no longer than 30 days. The voting period must be stated in the proposal in calendar days. The voting period begins on the day the pull request is opened for review and ends at midnight [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth) on the last day of the voting period. Proposals can be extended for no longer than 30 days total. Extensions can be made before the end of the current voting period by the original submitter, their direct representative, or one of the ART chairs by updating the pull request with a note stating the number of calendar days for the extension. **If approval is not reached within the voting period deadline, the proposal is rejected.**
```

Comment:
What, actually, does "direct representative" mean? There's no such concept in GitHub (or git, for that matter) ... what burden, specifically, does this place on reviewers or the chairs who have to evaluate the sequence of operations? How is anyone supposed to know whether some random person modifying the PR is a "direct representative"?

Reply:
Unless it has a standard definition and some mechanism to check/enforce, yeah, it's probably more confusing than useful.